src/lava/magma/runtime/message_infrastructure/CMakeLists.txt

```diff
@@ -26,6 +27,33 @@ set(MESSAGE_INFRASTRUCTURE_SRCS
 
 add_library(message_infrastructure SHARED ${MESSAGE_INFRASTRUCTURE_SRCS})
 
+set(MSG_LOG_LEVEL 16 CACHE STRING "Default vaule: 16, error log only.")
+set(MSG_LOG_PRINT_MODE 1 CACHE STRING "Default value: 1, print onto console only.")
```

Comment:
add instructions about `MSG_LOG_LEVEL` and `MSG_LOG_PRINT_MODE` into [run_example.md](https://github.com/lava-nc/lava/blob/messaging_refactor_release2nd/src/lava/magma/runtime/message_infrastructure/docs/run_example.md).